tests/ctst/features/crrReplicationS3utils.feature

```diff
@@ -2,15 +2,24 @@ Feature: Replication
     This feature tests replication using the script crrExistingObjects.js from S3utils,
     that replicates objects created before the replication was enabled on source the bucket.
 
+    @2.12.0
+    @PreMerge
+    @ReplicationTest
+    Scenario Outline: Objects created before setting up replication should not be replicated automatically
+        Given an existing bucket "source-bucket-0" "with" versioning, "without" ObjectLock "without" retention mode
+        And an object "source-object-1" that "exists"
+        And a replication configuration to "awsbackendmismatch" location
+        Then the object replication should "never happen" within 30 seconds
+
     @2.12.0
     @PreMerge
     @ReplicationTest
     Scenario Outline: Replicate objects created before creating the replication rule
-        Given an existing bucket "source-bucket" "with" versioning, "without" ObjectLock "without" retention mode
+        Given an existing bucket "source-bucket-1" "with" versioning, "without" ObjectLock "without" retention mode
         And an object "source-object-1" that "exists"
         And a replication configuration to "awsbackendmismatch" location
         When the job to replicate existing objects with status "NEW" is executed
-        Then the object should eventually "be" replicated
+        Then the object replication should "succeed" within 300 seconds
```

Comment:
For consistency if we expect a replication to succeed up to 300s, we should do the same for the "never happen" case, no?
But as pointed out this would artificially make tests very long. If we thus think 30s is enough, we should probably use it here as well.

Another approach for such "test to see if this never happens" would be to look more in depth at the system to see if the old object was scanned after enabling the replication configuration. This allows to have a condition to stop on. But this is probably not realistic at the end-to-end level. Do we think the scenario `Objects created before setting up replication should not be replicated automatically` can only be tested at the Zenko level, and not Backbeat as a functional test?